contracts/scripts/bridge/standard_bridge_deposit_erc20.ts

```diff
@@ -0,0 +1,113 @@
+import { ethers } from "hardhat";
+import { getSignersAndContracts, getStorageKey } from "./utils";
+
+async function main() {
+  const {
+    l1Provider,
+    l1Bridger,
+    l2Bridger,
+    l2Relayer,
+    l1Portal,
+    l2Portal,
+    l1StandardBridge,
+    l2StandardBridge,
+    l1Oracle,
+  } = await getSignersAndContracts();
+
+  const TestTokenFactory = await ethers.getContractFactory(
+    "TestToken",
+    l1Bridger
+  );
+  const l1Token = await TestTokenFactory.deploy();
+  const l1BalanceStart = await l1Token.balanceOf(l1Bridger.address);
+
+  const MintableERC20FactoryFactory = await ethers.getContractFactory(
+    "MintableERC20Factory",
+    l2Relayer
+  );
+  const mintableERC20Factory = await MintableERC20FactoryFactory.deploy(
+    l2StandardBridge.address
+  );
+  const deployTx = await mintableERC20Factory.createMintableERC20(
+    l1Token.address,
+    "TestToken",
+    "TT"
+  );
+  const deployTxWithLogs = await deployTx.wait();
+  const deployEvent = await mintableERC20Factory.interface.parseLog(
+    deployTxWithLogs.logs[0]
+  );
+  const l2TokenAddr = deployEvent.args.localToken;
+
+  const MintableERC20Factory = await ethers.getContractFactory(
+    "MintableERC20",
+    l2Relayer
+  );
+  const l2Token = MintableERC20Factory.attach(l2TokenAddr);
+
+  const approveTx = await l1Token.approve(
+    l1StandardBridge.address,
+    l1BalanceStart
+  );
+  await approveTx.wait();
+
+  const bridgeTx = await l1StandardBridge.bridgeERC20(
+    l1Token.address,
+    l2Token.address,
+    l1BalanceStart,
+    200_000,
+    []
+  );
+  const txWithLogs = await bridgeTx.wait();
+  const l1BalanceEnd = await l1Token.balanceOf(l1Bridger.address);
+
+  const initEvent = await l1Portal.interface.parseLog(txWithLogs.logs[3]);
+  const crossDomainMessage = {
+    version: 1,
```

Comment:
any reason we're using verison 1 instead of 0?

Reply:
no reason, currently any version number will work. changed to 0 to avoid confusion